main.sh

```diff
@@ -245,7 +230,7 @@ fi
 info "__file: ${__file}"
 info "__dir: ${__dir}"
 info "__base: ${__base}"
-info "__os: ${__os}"
+info "OSTYPE: ${OSTYPE}"
```

Comment:
This is the line I'm talking about. My Github iPhone  app may have botched the line numbers.